components/eamxx/src/physics/shoc/impl/shoc_main_impl.hpp

```diff
@@ -119,6 +119,7 @@ void Functions<S,D>::shoc_main_internal(
   // Output Variables
   Scalar&                      pblh,
   const uview_1d<Spack>&       shoc_ql2,
+  const uview_1d<Spack>&       tkh_out,
```

Comment:
We can just call this `tkh` and remove the other `tkh` from the workspace manager. Then no need to copy down below from `tkh`->`tke_out`.